Automattic/woocommerce-payments
# Add/1427 style woopay express checkout button

#### Changes proposed in this Pull Request

This PR adds styling for the WooPay Express Checkout button as well as converting it to a React component. This was done because we will need a React component for blocks checkout and we wanted only **one** instance of the button that needs to be maintained (as opposed to having a static HTML button in addition to the React component).

#### Testing instructions

* Install this PR.
* Make sure the latest dev tools plugin is installed.
* Go to the dev tools page and click the Enable the WooPay Express Checkout button. checkbox and save the settings.
* Go to the WooCommerce Payments settings screen and click "Customize" next to WooPay under "Express Checkouts."
* Make changes to the button's settings.
* View the button on the front end of the site (product, cart, and shortcode checkout pages) and make sure that the button appears styled according to the designs in Figma.

-------------------


**Post merge**
